lib/tower.ex

```diff
@@ -455,64 +455,46 @@ defmodule Tower do
     rescue
       e ->
         Event.from_exception(e, __STACKTRACE__, metadata: %{test: true})
-        |> report_event(async: false)
+        |> report_event()
 
         :ok
     end
   end
 
-  defp report_event(%Event{} = event, options \\ []) do
-    async = Keyword.get(options, :async, true)
+  def async(fun) do
+    Tower.TaskSupervisor
+    |> Task.Supervisor.start_child(fun)
```

Comment:
If this is going to be used in the reporter packages, need to evaluate if we prefer having linked tasked instead, by default.

Reply:
Not exposing for now

Reply:
👍🏼

Reply:
Reporters can still call/use `Tower.TaskSupervisor` directly.